shared/profile/user/index.js

```diff
@@ -394,13 +394,11 @@ const styles = Styles.styleSheetCreate({
     },
     isElectron: {
       borderBottomStyle: 'solid',
-      paddingBottom: Styles.globalMargins.tiny,
-      paddingLeft: Styles.globalMargins.small,
-      paddingRight: Styles.globalMargins.small,
-      paddingTop: Styles.globalMargins.medium,
+      height: 40,
```

Comment:
Is there any place to put these constants rather than hard-coding them?

Reply:
Maybe put it above below `headerHeight`? Also this is `globalMargins.large`; might as well use it in case we do zooming using that in the future?

Reply:
Oh nvm Cécile told me we shouldn't use global margins for dimensions. So LGTM as-is!

Reply:
I considered putting it near headerHeight, but it seemed more confusing since the desktop/mobile dimensions are pretty different. Leaving as is!